src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (a) => {
+    if (typeof a === 'undefined') {
```

Comment:
It's more idiomatic in JavaScript to check for 'undefined' using 'a === undefined' or simply 'a == null' if you want to check for both 'null' and 'undefined'.